pkg/virt-api/webhooks/validating-webhook/admitters/vmi-update-admitter.go

```diff
@@ -73,13 +74,13 @@ func admitVMILabelsUpdate(
 	ar *v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
 
 	// Skip admission for internal components
-	allowed := getAllowedServiceAccounts()
+	allowed := GetAllowedServiceAccounts()
 	if _, ok := allowed[ar.Request.UserInfo.Username]; ok {
 		return nil
 	}
 
-	oldLabels := filterKubevirtLabels(oldVMI)
-	newLabels := filterKubevirtLabels(newVMI)
+	oldLabels, _ := FilterKubevirtLabels(oldVMI.ObjectMeta.Labels)
+	newLabels, _ := FilterKubevirtLabels(newVMI.ObjectMeta.Labels)
 
 	if !reflect.DeepEqual(oldLabels, newLabels) {
```

Comment:
What happens here if the order is changed?

Reply:
@xpivarc Since it's a map the order should not affect the validation. Labels are the keys in these maps.